packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -207,6 +212,23 @@ export async function queryAndProcessDocuments({
     });
 
     log(`Finished processing ${fhirDocRefs.length} documents.`);
+
```

Comment:
Patient discovery shouldnt be intertwined with DQ

Reply:
No longer relevant.